automatic/jenkins/update.ps1

```diff
@@ -0,0 +1,42 @@
+#requires -Modules au, Wormies-AU-Helpers
+param(
+    # The URL to search for information on the latest release
+    $ReleasePage = "https://www.jenkins.io/download/",
+
+    # The regex used to search for the latest version on the ReleasePage
+    $ReleaseRegex = "Download Jenkins`n(?<Version>[\d\.]+)`nLTS",
+
+    # The base of the download URL
+    $DownloadURL = "https://mirrors.jenkins-ci.org/windows-stable/"
+)
+
+function global:au_BeforeUpdate {
+    Get-RemoteFiles -Purge -NoSuffix
+}
+
+function global:au_SearchReplace {
+    @{
+        ".\legal\VERIFICATION.txt" = @{
+            "^(?<UrlHeading>\W+- Direct: <.+)\/(?<PACKAGEVERSION>.+)/jenkins.msi>" = "`$1/$($Latest.Version)/jenkins.msi>"
+            "^(?<ChecksumTypeHeading>\W+- ChecksumType:)(?<CHECKSUMTYPE>.+)"       = "`$1 $($Latest.ChecksumType64)"
+            "^(?<ChecksumHeading>\W+- Checksum:)(?<CHECKSUM>.+)"                   = "`$1 $($Latest.Checksum64)"
```

Comment:
Use a similar regex that we use for other packages: https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/7zip.install/update.ps1#L14-L20